ext/node/polyfills/_events.mjs

```diff
@@ -469,7 +468,7 @@ function _addListener(target, type, listener, prepend) {
       w.emitter = target;
       w.type = type;
       w.count = existing.length;
-      nodeGlobals.process.emitWarning(w);
+      globalThis.process.emitWarning(w);
```

Comment:
You marked `process` as `writeable` in the file below, so the code here might start failing. I think we need to capture reference to process here?